Recommend Chocolately rsvg-convert (#1330)
wpilibsuite/frc-docs: source/docs/contributing/frc-docs/build-instructions.rst

Windows
^^^^^^^

> .. note:: MikTeX and ``rsvg-convert`` is not required for building HTML, it is only required for Windows PDF builds.

Reviewer: ```suggestion
.. note:: MikTeX and ``rsvg-convert`` are not required for building HTML, they are only required for Windows PDF builds.
```
